pkg/controller/controlplane/valuesprovider.go

```diff
@@ -995,7 +990,6 @@ func (vp *valuesProvider) addCSIManilaValues(
 	values["openstack"] = map[string]interface{}{
 		"availabilityZones":           vp.getAllWorkerPoolsZones(cluster),
 		"shareNetworkID":              shareNetworkID,
-		"shareClient":                 infrastructure.WorkersCIDR(infraConfig),
```

Comment:
How can the "shareClient" be set if we remove it from here?

Reply:
We don't set it basically. The shareClient constricts who can access the share, for which at the moment we have no strict need to do, it even restrict customer use cases. The template doesn't necessarily  need to change, we just choose not to configure this.